### DISCUSSION TITLE
"Extract into function" puts extra "\" in generated function only in WSL

### DISCUSSION THREAD

**Discussion by elithrade** [mrcjkb/rustaceanvim#332] [Q&A]
Hi,

I have a strange problem I cannot figure out why.
I have the following code:
```
fn main() {
    let n = 100;
    for i in 0..n {
        println!("{}", i);
    }
}
```
I select the for loop and bring up Lsp code actions and select "Extract into function" the generated function looks like this:
```
fn main() {
    let n = 100;
    fun_name(n);
}

fn fun_name(n: i32) {
    for i in 0..n {
        println!("{\}", i);
    \}
\}
```
![image](https://github.com/mrcjkb/rustaceanvim/assets/10831354/859de321-95ba-4df8-9b2b-55db15fcde83)
![image](https://github.com/mrcjkb/rustaceanvim/assets/10831354/63004fa8-4e68-4f37-8cab-350f175796ba)

I cannot figure out where those \ come from. My rust-anaylzer version is:
❯ rust-analyzer --version
rust-analyzer 1.77.1 (7cf61eb 2024-03-27)

I am using LazyVim with Rust language extras enabled.

I have the same setup on my Windows machine and it's working as expected. Any ideas? Looks like a WSL specific issue but I cannot figure out why, any helps appreciated.

**Comment by mrcjkb** [comment]
Duplicate of https://github.com/mrcjkb/rustaceanvim/issues/303

This is a neovim bug
